lib/faktory/job.rb

```diff
@@ -56,6 +56,10 @@ def initialize(opts)
         @opts = opts
       end
 
+      def set(opts)
+        self.class.new(@opts.merge(opts))
```

Comment:
Would it be reasonable to merge @opts directly rather than allocating a new instance?

Reply:
Yup. Fixed in 1b532f2.